woocommerce/woocommerce-android
# Feature/9080 share store my store tab

Part of: #9080 


### Description
As part of improving the product sharing experience, we are adding a new CTA to share user's store URL from my store tab. This button will be visible only if the store is `public` (has been launched). 

### Testing instructions

- Open the app using a site that hasn't been launched/published. Check the share button in the toolbar of my store tab is not displayed.
- Open a public/launched site and check the share button is visible. Clicking on it will open a bottom sheet of available apps to share the store url. The following event should be tracked when tapping on the share button: `🔵 Tracked dashboard_share_your_store_button_tapped.`


https://github.com/woocommerce/woocommerce-android/assets/2663464/45aa1cc3-12b3-4455-9e71-d18a6818448a
